README.md

```diff
@@ -44,6 +44,26 @@ For more info see: https://docs.ipfs.io/introduction/overview/
 
 ## Install
 
+Rust IPFS depends on `protoc` and `openssl`.
+
+### Dependencies
+
+First, install the dependencies.
+
+With apt:
+
+```bash
+# apt-get install protobuf-compiler libssl-dev
```

Comment:
I assume this is hash to symbolize the root prompt? I haven't used root shell for so long time :)

Reply:
You assume correctly :D